gsplat/__init__.py

```diff
@@ -1,166 +1,113 @@
-from typing import Any
-import torch
-from .project_gaussians import project_gaussians
-from .rasterize import rasterize_gaussians
-from .utils import (
-    map_gaussian_to_intersects,
-    bin_and_sort_gaussians,
-    compute_cumulative_intersects,
-    compute_cov2d_bounds,
-    get_tile_bin_edges,
-)
-from .sh import spherical_harmonics
-from .version import __version__
 import warnings
 
+from .rendering import rasterization
+from ._helper import rasterization_legacy_wrapper, rasterization_inria_wrapper
+from .version import __version__
```

Comment:
the version should probably be bumped from 0.1.11 to 1.0.0

Reply:
done